media/transcoder/src/main/java/com/example/transcoder/CreateJobWithAnimatedOverlay.java

```diff
@@ -0,0 +1,144 @@
+import com.google.cloud.video.transcoder.v1beta1.VideoStream;
+import com.google.protobuf.Duration;
+import java.io.IOException;
+
+public class CreateJobWithAnimatedOverlay {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "my-project-id";
+    String location = "us-central1";
+    String inputUri = "gs://my-bucket/my-video-file";
+    String overlayImageUri = "gs://my-bucket/my-overlay-image.jpg"; // Must be a JPEG
+    String outputUri = "gs://my-bucket/my-output-folder/";
+
+    createJobWithAnimatedOverlay(projectId, location, inputUri, overlayImageUri, outputUri);
+  }
+
+  // Creates a job from an ad-hoc configuration and adds an animated overlay to it.
+  public static void createJobWithAnimatedOverlay(
+      String projectId, String location, String inputUri, String overlayImageUri, String outputUri)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (TranscoderServiceClient transcoderServiceClient = TranscoderServiceClient.create()) {
+
+      VideoStream videoStream0 =
+          VideoStream.newBuilder()
+              .setCodec("h264")
+              .setBitrateBps(550000)
+              .setFrameRate(60)
+              .setHeightPixels(360)
+              .setWidthPixels(640)
+              .build();
+      AudioStream audioStream0 =
+          AudioStream.newBuilder().setCodec("aac").setBitrateBps(64000).build();
+      Overlay.Image overlayImage =
+          Overlay.Image.newBuilder()
+              .setUri(overlayImageUri)
+              .setResolution(NormalizedCoordinate.newBuilder().setX(0).setY(0).build())
+              .setAlpha(1)
+              .build();
+      Overlay.Animation animationFadeIn =
+          Animation.newBuilder()
+              .setAnimationFade(
+                  AnimationFade.newBuilder()
+                      .setFadeType(FadeType.FADE_IN)
+                      .setXy(NormalizedCoordinate.newBuilder().setX(0.5).setY(0.5).build())
+                      .setStartTimeOffset(Duration.newBuilder().setSeconds(5).build())
+                      .setEndTimeOffset(Duration.newBuilder().setSeconds(10).build())
+                      .build())
+              .build();
+      Overlay.Animation animationFadeOut =
+          Animation.newBuilder()
+              .setAnimationFade(
+                  AnimationFade.newBuilder()
+                      .setFadeType(FadeType.FADE_OUT)
+                      .setXy(NormalizedCoordinate.newBuilder().setX(0.5).setY(0.5).build())
+                      .setStartTimeOffset(Duration.newBuilder().setSeconds(12).build())
+                      .setEndTimeOffset(Duration.newBuilder().setSeconds(15).build())
+                      .build())
+              .build();
```

Comment:
This is a really large chunk of setup/boilerplate code. Maybe add newlines between each step and a comment explaining what the step is doing?

Reply:
Done